gating/update_dependencies/run

```diff
@@ -34,10 +34,20 @@ pushd $rpc_maas_dir
   #  or that start with 9 or 10 are filtered out.
   latest_maas_tag="$(git tag -l |grep -v '[a-zA-Z]\|^\(9\.\|10\.\)' |sort -n |tail -n 1)"
 popd
-# Insert latest maas tag into rpcd/etc/openstack_deploy/user_rpco_variables_defaults.yml
-#  Insert current SHA into functions.sh
-sed -i '/^maas_release:/ c\
-maas_release: "'${latest_maas_tag}'"' etc/openstack_deploy/group_vars/all/release.yml
+
+# Write to the rpc-release file
+python <<EOC
+import yaml
+release_file = "${GATING_PATH}/../../playbooks/vars/rpc-release.yml"
+with open(release_file) as f:
+  x = yaml.safe_load(f.read())
```

Comment:
only two spaces? for shame (nit)

Reply:
yes, two spaces because I'm using a heredoc and I wanted to keep it compressed.